clowdapp.yaml

```diff
@@ -9,8 +9,13 @@ objects:
       name: xjoin-api-gateway
     spec:
       envName: ${ENV_NAME}
+      testing:
+        iqePlugin: xjoin
       dependencies:
         - xjoin-apicurio
+      optionalDependencies:
+        - xjoin-search
+        - host-inventory
```

Comment:
Also wondering with host-inventory is required? The purpose of xjoin.v2 is to make xjoin generic so it shouldn't depend on host-inventory.

Reply:
Same answer as above. It's only optional, which means it's only needed for testing.